_posts/2023-09-22-licensing-update.md

```diff
@@ -0,0 +1,288 @@
+suitability of [LGPL v3+][LGPL v3], and the current status of the MDAnalysis [GPL v2+][GPL v2] package license, so we
+decided to pause the relicensing process whilst we worked with our legal counsel to find an appropriate path forward.
+
+After careful consideration we have decided to aim for a license change to the [LGPL v2.1+][LGPL v2.1] via interim
+[package license](#package-license) changes to [GPL v3+][GPL v3] and [LGPL v3+][LGPL v3].
+
+In this post we will provide our rationale for this change, alongside key information for how we will be
+enacting these changes.
+
+**License changes will affect how users and developers will interact with the MDAnalysis code base going
+forward. We urge all members of the community to read this post and [get in touch](#package-license) with
+the MDAnalysis core developer team should you have any questions or concerns**.
+
+## Rationale for our change in target license
+
+*Table 1: MDAnalysis’ view of the dynamic linking license compatibility matrix. License pairs
+detailed with a compatibility of “No” have claimed incompatibilities between their license terms.
+License pairs detailed as “Yes ([copyleft](#copyleft])” are compatible but may incur copyleft through the
+interacting GNU GPL license. License pairs detailed with the compatibility “Yes” are, to our knowledge,
+fully compatible without incurring any [copyleft](#copyleft) penalty.*
+
+| License         | [GPL v2][]          | [GPL v3][]          | [LGPL v2.1][]       | [LGPL v3][]         | [Apache 2.0]()     | [MIT][]            | [BSD 3.0][]        |
+|-----------------|---------------------|---------------------|---------------------|---------------------|--------------------|--------------------|--------------------|
+| [GPL v2][]      | Yes                 | No \[1\]            | Yes ([copyleft]())  | No \[1\]            | No                 | Yes ([copyleft]()) | Yes ([copyleft]()) |
+| [GPL v3][]      | No \[1\]            | Yes                 | No \[2\]            | Yes ([copyleft]())  | Yes ([copyleft]()) | Yes ([copyleft]()) | Yes ([copyleft]()) |
+| [LGPL v2.1][]   | Yes ([copyleft]())  | No                  | Yes                 | No \[2\]            | No                 | Yes                | Yes                |
+| [LGPL v3][]     | No \[1\]            | Yes ([copyleft]())  | No \[2\]            | Yes                 | Yes                | Yes                | Yes                |
+| [Apache 2.0][]  | No                  | Yes ([copyleft]())  | No                  | Yes                 | Yes                | Yes                | Yes                |
+| [MIT][]         | Yes ([copyleft]())  | Yes ([copyleft]())  | Yes                 | Yes                 | Yes                | Yes                | Yes                |
+| [BSD 3.0][]     | Yes ([copyleft]())  | Yes ([copyleft]())  | Yes                 | Yes                 | Yes                | Yes                | Yes                |
```

Comment:
I don't think the `[copyleft]()` in this table will end up linking anywhere - might need to be `[copyleft](#copyleft)`?